src/server/server_family.cc

```diff
@@ -1970,6 +1970,7 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
     append("total_net_input_bytes", conn_stats.io_read_bytes);
     append("connection_migrations", conn_stats.num_migrations);
     append("total_net_output_bytes", reply_stats.io_write_bytes);
+    append("rdb_save_usec", m.coordinator_stats.rdb_save_usec);
```

Comment:
I think maybe we can expose here avg time and not sum, maybe it will be more helpful to understand if there is a problem around writing to sink

Reply:
I thought about that originally. I think that a plain average can hide issues, especially after a long time in which the server is running, changing the average would take significant delays.
On the other hand, the approach here with total, if sampled over time, should show us when things spiked.
Maybe the best approach would be to add both count and the sum we have here.
wdyt?

Reply:
ok sure